### DISCUSSION TITLE
whats to do???

### DISCUSSION THREAD

**Discussion by NSH531** [PowerShell/PowerShell#18718] [General]
### Summary of the new feature / enhancement

Get-Certificate -Template SslWebServer -Credential $up -DnsName "we-can.sharefileftp.com" -CertStoreLocation cert:\LocalMachine\My -Url "ftp://we-can.sharefileftp.com:990/" 
cmdlet Get-Credential at command pipeline position 1
Supply values for the following parameters:
Get-Certificate : CX509EnrollmentPolicyWebService::Initialize: The parameter is incorrect. 0x80070057 (WIN32: 87 ERROR_INVALID_PARAMETER)
At line:5 char:1
Get-Certificate -Template SslWebServer -Credential $up -DnsName "we-can.sharefileftp.com" -CertStoreLocation cert:\LocalMachine\My -Url "ftp://we-can.sharefileftp.com:990/" 
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Get-Certificate], Exception
    + FullyQualifiedErrorId : System.Exception,Microsoft.CertificateServices.Commands.GetCertificateCommand

**Comment by TheOriginalPiff** [comment]
Business ContinuancySent from my iPhoneOn Dec 1, 2022, at 7:48 AM, NSH531 ***@***.***> wrote:﻿
Summary of the new feature / enhancement
Get-Certificate -Template SslWebServer -Credential $up -DnsName "we-can.sharefileftp.com" -CertStoreLocation cert:\LocalMachine\My -Url "ftp://we-can.sharefileftp.com:990/"
cmdlet Get-Credential at command pipeline position 1
Supply values for the following parameters:
Get-Certificate : CX509EnrollmentPolicyWebService::Initialize: The parameter is incorrect. 0x80070057 (WIN32: 87 ERROR_INVALID_PARAMETER)
At line:5 char:1

Get-Certificate -Template SslWebServer -Credential $up -DnsName "we-c ...

  + CategoryInfo          : NotSpecified: (:) [Get-Certificate], Exception
  + FullyQualifiedErrorId : System.Exception,Microsoft.CertificateServices.Commands.GetCertificateCommand





Proposed technical implementation details (optional)
No response

**Comment by NSH531** [comment]
@TheOriginalPiff what's the problem 
I don't understand this

nate

**Comment by kilasuit** [comment]
@NSH531 this is better asked in either a forum or a discussion as it is not an issue with PowerShell but how you are using it. I'd recommend signing up to the [PowerShell Discord](https://aka.ms/psdiscord) where you may get more help. However I am going to convert this to a discussion as this isn't an issue
